wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,303 @@
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | *2007-12-24* | 2008-09-07 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | 2008-09-07 | 2008-09-07 |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2008-09-07 | 2008-10-07 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-10-07 | 2008-12-11 |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 2008-12-11 | *2009-01-19* |
+
+## 2009
+
+| Player | From | To |
+| --: | :-- | :-- |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | *2008-12-11* | 2009-01-19 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-01-19 | March 2009 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | March 2009 | March 2009 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | March 2009 | April 2009 |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | April 2009 | April 2009 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | April 2009 | June 2009 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | June 2009 | July 2009 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | July 2009 | July 2009 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | July 2009 | July 2009 |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | July 2009 | September 2009 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | September 2009 | November 2009 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | November 2009 | *2010-02-07* |
+
+## 2010
+
+| Player | From | To |
+| --: | :-- | :-- |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *November 2009* | 2010-02-07 |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2010-02-07 | 2010-09-08 |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | 2010-09-08 | September 2010 |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | September 2010 | October 2010 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | October 2010 | November 2010 |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | November 2010 | November 2010 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | November 2010 | December 2010 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | December 2010 | *January 2011* |
+
+## 2011
+
+| Player | From | To |
+| --: | :-- | :-- |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *December 2010* | January 2011 |
+| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | January 2011 | January 2011 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | January 2011 | January 2011 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | January 2011 | February 2011 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | February 2011 | 2011-04-23 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | 2011-04-23 | 2011-05-31 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2011-05-31 | 2011-06-05 |
```

Comment:
```suggestion
| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | 2011-04-23 | 2011-05-18 |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2011-05-18 | 2011-06-05 |
```
- chocomint was already rank 1 earlier on 18 May as can be concluded from this archive: https://web.archive.org/web/20110518091909/http://osu.ppy.sh/p/playerranking/?m=0&s=2&r=&q=&f= (note that this is an archive of the leaderboard sorted by TOTAL score, not ranked score)

Reply:
Updated with https://github.com/ppy/osu-wiki/pull/4720/commits/b9f1ab80d6cb9c10aaabdd4b370ffe02008be4b8, need to add links to sources column at somepoint